tox.ini

```diff
@@ -106,24 +106,24 @@ commands    =
                      {envlogdir}/coverage.xml
 
 [covcombine]
-description = Generate combined coverage reports with py27-test coverage merged
+description = Generate combined coverage reports with py27-cov coverage merged
 commands =
-    tox -e py27-test
+    tox -e py27-cov
```

Comment:
Instead of this change, I'll move the `coverage xml` step to the test execution,
otherwise this fails on changes specific to Python2.7 or Python3.